community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/slurmsync.py

```diff
@@ -506,9 +506,13 @@ def sync_maintenance_reservation(lkp: util.Lookup) -> None:
           continue
         else:
           del_reservation.add(res_name)
-          create_reservation_map[res_name] = (node, start_time)
+          nodeset = lkp.node_nodeset(node)
```

Comment:
* Let's lookup nodeset once, e.g. at line 503.
* `lkp.node_nodeset` can throw, let's do it safely (e.g. wrap in try/catch).

Reply:
outdated.